app/views/staff/shared/activities/_activity.html.haml

```diff
@@ -395,7 +395,7 @@
         = activity_presenter.policy_marker_nutrition
       %dd.govuk-summary-list__actions
         - if policy(activity_presenter).update? && step_is_complete_or_next?(activity: activity_presenter, step: :policy_markers)
-          = a11y_action_link(t("default.link.#{activity_presenter.call_to_action(:policy_marker_nutrition)}"), activity_step_path(activity_presenter, :policy_markers), t("summary.label.activity.policy_marker_nutrition"))
+          = a11y_action_link(t("default.link.#{activity_presenter.call_to_action(:policy_marker_nutrition)}"), activity_step_path(activity_presenter, :policy_markers, anchor: "nutrition"), t("summary.label.activity.policy_marker_nutrition"))
```

Comment:
Do you think that the inclusion of these new `anchor` attributes in the links should be described in a spec to prevent regressions?

Reply:
Possibly - it's not a "core" feature IMO, but if accepted (I went a bit over the scope of the card because it was a quality-of-life improvement for me, and I projected its need onto our users) then yes, we should also try to maintain it.